apps/studio/components/interfaces/Auth/Users/Users.utils.tsx

```diff
@@ -132,8 +133,15 @@ export function getDisplayName(user: User, fallback = '-'): string {
     first_name,
   } = user.raw_user_meta_data ?? {}
 
-  const last = familyName || family_name || surname || lastName || last_name
-  const first = givenName || given_name || firstName || first_name
+  const last =
+    (custom_claims as any)?.last_name ||
```

Comment:
i realised this custom_claims property exists for SAML users

Reply:
Good catch. It may not be called `last_name`/`first_name` every time though, so maybe if we can go through all the options like we normally do on `custom_claims` first, then callback to the normal variables?